test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -250,6 +251,25 @@ TEST_F(GrpcHttp1BridgeFilterTest, HandlingBadGrpcStatus) {
   EXPECT_EQ("foo", response_headers.get_("grpc-message"));
 }
 
+TEST_F(GrpcHttp1BridgeFilterTest, HandlingNoTailerBadGrpcStatus) {
```

Comment:
A comment on top here saying something like `// Verifies that we convert grpc-status to the appropriates status code for a headers only response` might be helpful